subprojects/logging/src/main/java/org/gradle/internal/logging/serializer/ProgressStartEventSerializer.java

```diff
@@ -25,6 +25,8 @@
 
 public class ProgressStartEventSerializer implements Serializer<ProgressStartEvent> {
```

Comment:
Thanks Adam, we have unit tests for serializing empty values [here](https://github.com/gradle/gradle/blob/master/subprojects/logging/src/test/groovy/org/gradle/internal/logging/serializer/ProgressStartEventSerializerTest.groovy#L62-L81), but you reminded me to add unit tests for OperationIdentifier, which I did in 321389e. Any other specific cases you see missing?